packages/react/src/components/breadcrumb/Breadcrumb.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { Breadcrumb } from './Breadcrumb';
+
+describe('<Breadcrumb /> spec', () => {
+  const defaultList = [
+    { title: 'Home', path: '/' },
+    { title: 'Level 1', path: '/level1' },
+    { title: 'Level 2', path: '/level2' },
+    { title: 'Level 3', path: '/level3' },
+    { title: 'Current', path: null },
+  ];
+  it('renders the component', () => {
+    const { asFragment } = render(<Breadcrumb list={defaultList} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+  it('should not have basic accessibility issues', async () => {
+    const { container } = render(<Breadcrumb list={defaultList} />);
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+  it('renders all items. Last item with a path (#3) is also in mobile view and therefore rendered twice.', async () => {
```

Comment:
Could this be simplified? The name is a bit long and sounds like it's testing many things at once. Could they be divided into multiple tests? The mobile view can be tested by changing the global.innerWidth, I fiddled with that in my [mediaQuery hooks PR tests.](https://github.com/City-of-Helsinki/helsinki-design-system/pull/960/files#diff-3dc9182200c0fa286be8d80e6ca741f53bafcdb870fde5ce6ab0bf0182bb6ba4)

Reply:
Name is now simplified. Maybe it was misinformative. It tests the mobile view has correct item picked from the list.

Mobile/desktop views are shown via css, so I don't think there is a need to test breakpoints in Jest.